pkg/ccl/streamingccl/streamingest/stream_replication_e2e_test.go

```diff
@@ -432,3 +434,47 @@ func TestTenantStreamingPauseOnError(t *testing.T) {
 	require.True(t,
 		streamIngestionStats(t, c.destSysSQL, ingestionJobID).IngestionProgress.StartTime.IsEmpty())
 }
+
+func TestTenantStreamingCutoverOnSourceFailure(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	// TODO(casper): now this has the same race issue with
+	// TestPartitionedStreamReplicationClient, please fix them together in the future.
+	skip.UnderRace(t, "disabled under race")
```

Comment:
Is it failing with #83694 or something else?  If it is that issue, can we use `UnderRaceWithIssue`